.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug Report
+about: Submit a bug report if something isn't working as expected.
+title: ""
+labels: bug, triage
+assignees: ""
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
Worth sharing an example? Perhaps an exemplary bug/issue from the past?

Reply:
Same as above.  I do want to sync these issue templates across our three project repos, but we could add repo-specific example links once we have some to choose from.